thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/detector/email/filter/AlertFilterFactory.java

```diff
@@ -0,0 +1,87 @@
+package com.linkedin.thirdeye.detector.email.filter;
+
+import com.linkedin.thirdeye.datalayer.pojo.AnomalyFunctionBean;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Collections;;
+import java.util.Map;
+import java.util.Properties;
+import org.apache.commons.io.IOUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+
+public class AlertFilterFactory {
+  private static Logger LOGGER = LoggerFactory.getLogger(AlertFilterFactory.class);
+  public static final String FILTER_TYPE_KEY = "type";
+  public static final String ALPHA_BETA_ALERT_FILTER = "alpha_beta";
```

Comment:
After changing the props to contain all types of alert filter, the two types here can also be removed.